mongodb-odbc-driver/bin/build-mongo-odbc.sh

```diff
@@ -11,6 +11,15 @@
     rm -rf "$BUILD_DIR"
     mkdir -p "$BUILD_DIR"
 
+    if [ "$PLATFORM" = macos ]; then
+        cd "$BUILD_DIR"
+        # unfortunately, we unzip this in two places (libmongosql and here)
```

Comment:
We can move downloading openssl as a pre func in evg.yml to avoid this, or did I miss something?

Reply:
The goal is to avoid doing anything in the evg file.

Reply:
Why is it that we don't want to change the evg file?
I count 4 download and unzip. build-mongo-odbc.sh and create-dmg.sh are even part func for the same task. So when running the 'build' task, openssl is downloaded twice. Same for 'run local integration tests' function, start-sqlproxy.sh and run-local-integration-tests.sh are both called in this function (on mac).
What about :
- Adding OPENSSL_DIR and OPENSSL_PATH (looks like you have the same variable name but point to either the root dir or the lib dir of openssl) in PREPARE_SHELL.
- Creating a new download-openssl.sh script which download, unzip and set the correct pernission on the libs
- I would even create a func for it to use in 'build' before `- func: "build odbc driver`.
- Call the script where you need it